src/worker/network.rs

```diff
@@ -346,29 +347,38 @@ impl NetworkWorker {
 
         (iface, device, fd)
     }
+}
 
-    /// Map socket index to a local port and address. The address octets are
-    /// incremented every 100 sockets, whithin this interval the local port
-    /// is incremented.
-    fn get_local_addr_port(
-        &self,
-        addr: Ipv4Address,
-        index: usize,
-    ) -> (IpAddress, u16) {
-        // 254 (a2 octet) * 254 (a3 octet) * 100 (port)
-        // gives us maximum 6451600 connections that could be opened
-        let local_port = 49152 + (index % 100) as u16;
-        debug!("addr {}, index {}", addr, index);
-
-        let local_addr = IpAddress::v4(
-            addr.0[0],
-            addr.0[1],
-            (((index / 100) + 2) / 255) as u8,
-            (((index / 100) + 2) % 255) as u8,
-        );
-
-        (local_addr, local_port)
+/// Map socket index to a local port and address. The address octets are
+/// incremented every conns_per_addr sockets, whithin this interval the local port
+/// is incremented.
+fn get_local_addr_port(
```

Comment:
`self` was not used, so I removed it and moved this function outside of the class.